packages/InteractiveProfilingTool-Core.package/IPTTreeModel.class/instance/getChildren..st

```diff
@@ -0,0 +1,4 @@
+model access
+getChildren: aMessageTally
+
+	^ self getChildrenWithRelevantOverallPerformanceImpactFor: aMessageTally
```

Comment:
This method does only return Tallys for methods that took more than 1% of the overall runtime, right? If this is true, it should be documented somewhere, since I would expect something else when calling ```getChildren``` (I would expect all children).

Reply:
Also it seems like ```getChildrenWithRelevantOverallPerformaceImpactFor``` could just be integrated here and not in a seperate function?!

Reply:
I agree with you. Maybe we should name the method 'getReleveantChildren'. In this case the purpose from the getChildren method is more clear. To get just the relevant children of the *Messsage tally tree' that should be displayed in the PluggableTreeMorph. What do you think @linusha ?

In case of the 'getChildrenWithRelevantOverallPerformanceImpectFor' I think the separate function is helping a lot witch the readability. But an alternative could be a small comment in the Methode.

Reply:
Method `getChildrenWithRelevantOverallPerformaceImpactFor` was removed and the `hasChildren` and `getChildren` methods now have more descriptive names.